packages/webapp/src/routes/__tests__/land/legal-party-list.spec.js

```diff
@@ -0,0 +1,93 @@
+import { submitGetRequest } from '../helpers/server.js'
+import constants from '../../../utils/constants.js'
+const url = constants.routes.LEGAL_PARTY_LIST
+
+describe('url', () => {
```

Comment:
Probably want to describe `url` and not the string 'url'

Reply:
duh, yes ofcourse :)